catalog/spatial/geocoding/spatial-geocoding-geocoder/src/test/java/org/codice/ddf/spatial/geocoder/GazetteerGeoCoderTest.java

```diff
@@ -182,8 +182,7 @@ public void testGetCountryCodeParseException() throws ParseException, GeoEntryQu
 
   @Test
   public void testGetCountryCodeInvalidCountryCode() throws ParseException, GeoEntryQueryException {
-    when(geoEntryQueryable.getCountryCode(TEST_POINT, 50))
-        .thenReturn(Optional.of("not a country code"));
+    when(geoEntryQueryable.getCountryCode(TEST_POINT, 50)).thenReturn(Optional.empty());
```

Comment:
This change should be reverted. The condition `when(geoEntryQueryable.getCountryCode(TEST_POINT, 50)).thenReturn(Optional.empty())` is already being tested in https://github.com/codice/ddf/pull/3232/files#diff-fccb280d010b95893afc72c8bd72ac83R158